scripts/start.py

```diff
@@ -52,6 +52,13 @@
     '--enable_console',
     help='optional; if specified, enables console.',
     action='store_true')
+_PARSER.add_argument(
+    '--disable_host_checking',
+    help=(
+        'optional; if specified, disables host checking so that the dev '
+        'server can be accessed by any device on the same network using the '
+        'host device\'s IP address.'),
```

Comment:
Can you please also add: "DO NOT use this flag if you're running on an untrusted network."

Reply:
Done